app.js

```diff
@@ -1,93 +0,0 @@
-if (process.env.NODE_ENV !== "production") {
-  require("dotenv").config();
-}
-
-const express = require("express");
-const app = express();
-const path = require ("path");
-const mongoose = require("mongoose");
-
-const Stock = require("./models/stock");
-const Count = require("./models/visitorCount");
-
-const dbUrl = process.env.DB_URL;
-
-mongoose.connect(dbUrl, { useNewUrlParser: true }, { useUnifiedTopology: true })
-    .then(() => {
-        console.log("MONGO CONNECTION OPEN")
-    })
-    .catch(err => {
-        console.log("MONGO CONNECTION ERROR")
-        console.log(err)
-    })
-
-app.set("views", path.join(__dirname,"views"));
-app.set("view engine", "ejs");
-
-app.use(express.urlencoded ({extended:true}));
-app.use(express.static(__dirname));
-
-
-function getRankings(prevStockList, currStockList) {
-  let rankingChanges = [];
-  let prevPositions = [];
-  let currInPrevPositions = [];
-
-  for (prevStock of prevStockList) {
-    prevPositions.push(prevStock["ticker"]);
-  }
-
-  for (let i=0; i<10; i++) {
-    currInPrevPositions.push(prevPositions.indexOf(currStockList[i]["ticker"])+1);
-  }
-
-  for (let i =0; i<10; i++) {
-    if (currInPrevPositions[i] == 0) {
-      rankingChanges.push({"ticker": currStockList[i]["ticker"]});   
-    }
-    else {
-      rankingChanges.push({"ticker": currStockList[i]["ticker"], "change": currStockList[i]["rank"]-currInPrevPositions[i]});  
-    }
-    
-  }
-  return rankingChanges;
-}
-
-app.get("/", async (req, res) => {
-  const latestCount = await Count.find().sort({_id: -1}).limit(1);
-  const visitorCount = latestCount[0].count + 1; 
-  const filter = {count: visitorCount - 1};
-  const update = {count : visitorCount};
-  let doc = await Count.findOneAndUpdate(filter, update, {upsert: true, sort: {created: -1}});
-
-  const latest = await Stock.find().sort({timeStamp:-1}).limit(1);
```

Comment:
Wrap this into another function called loadTicker or something to make this function really simple and easier to read.